CHANGES/7916.doc

```diff
@@ -0,0 +1 @@
+Updated contributing/Tests coverage section to show how we use ``codecov``. -- by :user:`Dreamsorcerer`
```

Comment:
I didn't update examples in this repo but essentially, it's nicer to have a trailing period when there's a single sentence and the byline is on separated from the paragraph.
```suggestion
Updated contributing/Tests coverage section to show how we use ``codecov`` -- by :user:`Dreamsorcerer`.
```
Additionally, you can use RST syntax to actually reference those sections so that they are clickable on the changelog page on RTD.